install

```diff
@@ -120,8 +120,8 @@ else
 fi
 
 if ! ask_Yn "Are you content with using nano as your 'EDITOR'?"; then
-  ask "What should we set 'EDITOR' to?"
-  sed -i s/export EDITOR=nano/export EDITOR=\""$REPLY"\"/ ~/.zshrc
+  ask "What should we set 'EDITOR' to? [command name only, not a path]"
+  sed -i '' 's/export EDITOR=nano/export EDITOR="'$REPLY'"/' ~/.zshrc 
```

Comment:
This surprizes me. This worked fine for me. Is there perhaps something special about your input?

Reply:
nope, it's just `vim`.
Perhaps you've got `gnu-sed` installed already? 

```
> which sed
/usr/bin/sed
> sed --help
sed: illegal option -- -
usage: sed script [-Ealnru] [-i extension] [file ...]
        sed [-Ealnu] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
`gnu-sed` is `gsed`, not `sed`. Let me have a look.

Reply:
Ok. I was wrong.